Installer/dolphin-emu-nogui.6

```diff
@@ -0,0 +1,56 @@
+.TH DOLPHIN-EMU-NOGUI 6 "March 4, 2016"
```

Comment:
I think in right in Data/ is ok (just don't put it in Sys). Once that's done all the Installer/dolphin-emu.* files should be moved as well.